src/easyreflectometry/sample/sample.py

```diff
@@ -69,7 +69,7 @@ def as_dict(self, skip: list = None) -> dict:
             skip = []
         this_dict = super().as_dict(skip=skip)
         for i, layer in enumerate(self.data):
-            this_dict['data'][i] = layer.as_dict()
+            this_dict['data'][i] = layer.as_dict(skip=skip)
```

Comment:
shouldn't you use `layer` rather than  the generic `data`?

Reply:
I agree but the `this_dict` is made by `super` from easyscience and I don't want to change that.